interfaces/cython/cantera/examples/reactors/surf_pfr.py

```diff
@@ -114,12 +114,13 @@
 
     if n % 10 == 0:
         print('  {0:10f}  {1:10f}  {2:10f}  {3:10f}'.format(
-            dist, *gas['CH4', 'H2', 'CO'].X))
+            dist, *r.thermo['CH4', 'H2', 'CO'].X))
 
     # write the gas mole fractions and surface coverages vs. distance
     output_data.append(
-        [dist, r.T - 273.15, r.thermo.P/ct.one_atm] + list(gas.X)
-        + list(surf.coverages)
+        [dist, r.T - 273.15, r.thermo.P/ct.one_atm]
```

Comment:
```suggestion
        [dist, r.T - 273.15, r.thermo.P / ct.one_atm]
```
Just a minor suggestion, as Cantera does use spaces in this scenario (and should be updated when a line is changed)